PeripheryFunctions/TS_LoadFeatures.m

```diff
@@ -0,0 +1,20 @@
+function TS_LoadFeatures(whatDataFile,TS_DataMatToLoad,TS_CalcTimeToLoad,TS_QualityToLoad,INP_opsToLoad,INP_tsToLoad)
+
```

Comment:
Can you add a brief description of this function? What's its purpose?

Reply:
Done in new commit